Assets/MixedRealityToolkit/Graph/Plugins/Newtonsoft.Json.dll.meta

```diff
@@ -0,0 +1,103 @@
+fileFormatVersion: 2
+guid: abc5aed9aec861241ac2df4d98e3d04b
```

Comment:
We're not going to be putting this dll into vNEXT. It causes too much trouble with other packages that also include it.

We should be using the `Unity.JsonUtility` where possible.

Reply:
Agreed!!